README.md

```diff
@@ -7,3 +7,11 @@ This is an example of a monorepo managed by [PDM](https://pdm.fming.dev).
 - `packages/pkg-core`
 - `packages/pkg-first`
 - `packages/pkg-second`
+
+
+## Note about `pdm instal
+
+
+If you change the dependencies in one of the subprojects, the `pdm install` command will not detect that the lock file needs to be re-generated.
+
+You will thus have to run `pdm lock` manually, before running `pdm install`. 
```

Comment:
A single command `pdm update` can do the trick

Reply:
How would you do this for one of the sub-projects ?

Reply:
Assume the subproject name stored in the parent `pyproject.toml` is `foo` and you do `pdm update foo`

Reply:
Thanks, that's good to know ! 

In my associated "real-world" case, the sub-dependency was in a group, but not the dev one. Running `pdm update sub_project` worked as expected though :) (without specifying any group)


```bash
>  pdm update sub_project
🔒 Lock successful
Synchronizing working set with resolved packages: 0 to add, 1 to update, 0 to remove
  ✔ Update sub-project 0.1.0+editable -> 0.1.0 successful
Installing the project as an editable package...
  ✔ Update project 1.29.0+editable -> 1.29.0+editable successful
```